Data Structures/Array Implementation/ArrayImplementation(Insert,Delete,Linear Search).cpp

```diff
@@ -0,0 +1,84 @@
+//A programme for basic Array Implmentation - Insertion, Deletion, Search (Linear).
+
+#include <stdio.h>
+
+  int linearsearch(int value, int n, int array[]) {
```

Comment:
Fix indentation here. Extra space in the beginning.

Reply:
Also, rename the functions as per naming conventions.

Reply:
How to rename the functions (as per naming conventions) ?

Reply:
Yes